lib/ansible/modules/cloud/linode/linode.py

```diff
@@ -41,12 +41,90 @@
      - To keep sanity on the Linode Web Console, name is prepended with LinodeID_
     default: null
     type: string
+  displaygroup:
+    description:
+     - Add the instance to a Display Group in Linode Manager
+    default: null
+    type: string
+    version_added: "2.2"
```

Comment:
We'll need to update the version_added to either 2.3 or 2.4 depending on whether we can get this in before 2.3's module freeze date.  (A couple weeks from now).